ui/src/layouts/log_in/entry_point.rs

```diff
@@ -264,7 +264,7 @@ pub fn Layout(cx: Scope, page: UseState<AuthPages>, pin: UseRef<String>) -> Elem
                         },
                         aria_label: "create-account-button".into(),
                         appearance: kit::elements::Appearance::Primary,
-                        icon: if *cmd_in_progress.get() {Icon::Loader} else {Icon::Check},
+                        loading:  if *cmd_in_progress.get() { true } else { false },
```

Comment:
```suggestion
                        loading:  cmd_in_progress.get(),
```

(might need to dereference it but this should be fine according to CI)